experimenter/experimenter/nimbus_ui/static/js/theme_utils.js

```diff
@@ -0,0 +1,25 @@
+import { Compartment } from "@codemirror/state";
+import { oneDark } from "@codemirror/theme-one-dark";
```

Comment:
shouldn't we should explicitly depend on this in package.json? likewise for codemirror/state